### DISCUSSION TITLE
Parse POST form

### DISCUSSION THREAD

**Discussion by cdtut** [honojs/hono#2464] [Q&A] [answered]
I POST a form and `await c.req.text()` show `a=a&b=b&b=c`. How to parse to object like `{ a: "a", b: ["b", "c"] }`?

**Comment by NicoPlyley** [comment, marked as answer]
Did you set your encoding to be plain text? By default it should be application/x-www-form-urlencoded so you can use `await c.req.parseBody()`

[Docs here](https://hono.dev/api/request#parsebody) on how you can access it. It seems to be close to how your looking for
